pkg/log/request.go

```diff
@@ -23,7 +23,7 @@ import (
 )
 
 // CorrelationIDHeaders are the headers whose values will be taken as a correlation id for incoming requests
-var CorrelationIDHeaders = []string{"X-Correlation-ID", "X-CorrelationID", "X-ForRequest-ID", "X-Request-ID"}
+var CorrelationIDHeaders = []string{"X-Correlation-ID", "X-CorrelationID", "X-ForRequest-ID", "X-Request-ID", "X-Vcap-Request-Id"}
```

Comment:
Is this the right order?
It seems this should be configurable for each deployment depending on the infrastructure.
I guess X-Vcap-* is relevant only for CF.

Reply:
What do you mean by order?
There is a FixIt item to allow this list to be extensible/replaced per deployment.